partner_event/models/event_registration.py

```diff
@@ -0,0 +1,60 @@
+
+# © 2014 Tecnativa S.L. - Pedro M. Baeza
+# © 2015 Tecnativa S.L. - Javier Iniesta
+# © 2016 Tecnativa S.L. - Antonio Espinosa
+# © 2016 Tecnativa S.L. - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class EventRegistration(models.Model):
+    _inherit = "event.registration"
+
+    partner_id = fields.Many2one(
+        ondelete='restrict',
+    )
+    attendee_partner_id = fields.Many2one(
```

Comment:
I think it would make sense to add a unique constraint per event/attendee_partner_id.

Reply:
Mm, I think that would crash `event_sale` since it creates the registrations first and then lets the user change them